src/test/regress/sql/insert_select_into_local_table.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE dist_table (a INT, b INT, c TEXT);
```

Comment:
Please add lines like these to the top (basically always a good idea, if you use a semi-random `next_shard_id`):
```sql
SET citus.shard_count = 4;
SET citus.next_shard_id TO 1234900;
```

Without it this command fails:
```
make install -j9 && make -C src/test/regress/ check-minimal EXTRA_TESTS='insert_select_into_local_table'
```